src/packages/log-viewer/workspace/views/overview/components/log-viewer-message-templates-overview.element.ts

```diff
@@ -54,11 +67,11 @@ export class UmbLogViewerMessageTemplatesOverviewElement extends UmbLitElement {
 	render() {
 		return html`
 			<uui-box headline="Common Log Messages" id="saved-searches">
-				<p style="font-style: italic;">Total Unique Message types: ${this._messageTemplates?.total}</p>
+				<p style="font-style: italic;">Total Unique Message types: ${this._total}</p>
```

Comment:
Do we have a localization key for this or is it new?

Reply:
This message exists in old Backoffice, but it doesn't seem like there is any localization keys for it. There seems to be more things missing for log viewer like "common log messages" and "number of errors", etc.